TestProjects/Playground/Playground.Core/ViewModels/RootViewModel.cs

```diff
@@ -86,6 +88,8 @@ protected override void ReloadFromBundle(IMvxBundle state)
 
         public IMvxAsyncCommand ShowMixedNavigationCommand { get; private set; }
 
+        public IMvxAsyncCommand ShowDictionarySheetCommand { get; private set; }
```

Comment:
Shouldn't this command be named `ShowDictionaryBindingCommand` instead of `ShowDictionarySheetCommand`?